resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -48,6 +65,61 @@
           description: |
             Sets the number of reads to mapping in each loop. Larger number results in program
             taking more memory. This is especially evident in paired-end mapping.
+        - name: spikein_name
+          label: Chromosome name of unmethylated control sequence
+          type: basic:string
+          default: '' 
```

Comment:
Why not `required: false` instead of setting the empty default?

Reply:
None is passed to walt process and then grep command fails.